services/orders/index.js

```diff
@@ -6,6 +6,7 @@ module.exports = (db) => {
   router.get('/orders', require('./getOrder')(db));
   router.post("/:order", require("./addToOrder")(db));
   router.get("/orders", authorization, require("./getOrdersInCart")(db));
+  router.get("/ordersByDate", authorization, require("./get-order-by-date")(db));
```

Comment:
Mal nombre para una ruta si estamos haciendo una API Rest, como sería un "filtrado" de todos los pedidos, lo suyo sería trabajar directamente sobre la ruta ya creada de `/orders`. No obstante podéis hacer una adicional pero deberíais cambiarle el nombre